pkg/provider/azure_vmss_cache.go

```diff
@@ -33,6 +34,11 @@ import (
 	"sigs.k8s.io/cloud-provider-azure/pkg/consts"
 )
 
+var (
+	// ErrorManagedByUnknown indicates an instance management type is unknown.
+	ErrorManagedByUnknown = errors.New("vm management type unknown")
```

Comment:
this var is not used?

Reply:
Thanks for catching this. Just removed it.